src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, String name, int radius) {
+        super(color, name);
+        this.radius = radius;
```

Comment:
The `name` parameter is being passed to the superclass constructor. Ensure that this is necessary, as the superclass might already handle the `name` property. If it's redundant, consider removing it.